app/assets/javascripts/tylium.js

```diff
@@ -30,6 +30,8 @@
 //= require tylium/plugins/jquery.treenav
 
 //= require tylium/behaviors
+//= require shared/behaviors
```

Comment:
any reason not to include this in the `shared/` block (around L26)?

Reply:
If EditorToolbar init's before textile it breaks.